bin/propolis-server/src/lib/stats/mod.rs

```diff
@@ -171,8 +188,13 @@ pub fn start_oximeter_server(
 pub(crate) fn create_kstat_sampler(
     log: &Logger,
     n_vcpus: u32,
+    n_interfaces: u32,
```

Comment:
Optional: instead of having bare u32s it might be a bit cleaner to have this function take the instance properties/spec as an argument and peel off the values it needs (instead of giving callers the chance to make a transposition mistake).